server/src/server.rs

```diff
@@ -960,32 +995,28 @@ impl<RpcMiddleware, HttpMiddleware> TowerService<RpcMiddleware, HttpMiddleware>
 	}
 }
 
-impl<RpcMiddleware, HttpMiddleware> hyper::service::Service<hyper::Request<hyper::Body>>
-	for TowerService<RpcMiddleware, HttpMiddleware>
+impl<B, RpcMiddleware, HttpMiddleware> Service<HttpRequest<B>> for TowerService<RpcMiddleware, HttpMiddleware>
```

Comment:
nit: Maybe instead of `B` we can call this `Body`? To keep consistency with `RpcMiddleware` and `HttpMiddleware`?